components/actions/ContactPerson.js

```diff
@@ -25,6 +25,10 @@ const Person = styled.div`
       border: 4px solid ${(props) => props.theme.brandDark};
     }
   }
+
+  &.leader-no-avatar {
+    border-left: 5px solid ${(props) => props.theme.brandDark};
+  }
```

Comment:
The old code goes against our conventions here, so this might be a good time to refactor. Rather than using css classes, we should pass props to styled components to avoid any global classes which accidentally pick up other styles. Styled components will automatically create scoped classnames for us and makes sure they're server side rendered.

That would look something like this (I haven't tested it):

```ts
const Person = styled.div<{ $isLeader?: boolean, $withoutAvatar?: boolean }>`
  display: flex;
  margin-top: 1em;
  padding-bottom: 1em;
  border-bottom: 2px solid ${(props) => props.theme.themeColors.light};
  img {
    border: 2px solid ${(props) => props.theme.themeColors.light};
  }

  ${({ $isLeader }) => $isLeader && `
    img {
      border: 4px solid ${(props) => props.theme.brandDark};
    }
  `}

  ${({ $isLeader, $withoutAvatar }) => $withoutAvatar && $isLeader && `
    border-left: 5px solid ${(props) => props.theme.brandDark};
  `}}
```

Then the styles can be simply controlled via props: `<Person $isLeader={leader} $withoutAvatar={...}>...`